src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -214,9 +214,13 @@ export const DownloadCartPage:React.FunctionComponent<DownloadCartPageProps> = (
                       </ul>
                     </Typography>
                     <span>
-                      <a className="highlight-link disabled"
+                      <a className="highlight-link"
+                        rel="noreferrer"
                         data-for="downloadProgrammaticallyTooltipId"
-                        data-tip="This feature is coming soon.<br />You can still download individual<br />files programmatically.">
+                        data-tip="This feature is coming soon.<br />Click here to learn how to download<br />programmatically using other methods"
+                        target="_blank"
+                        href="https://help.synapse.org/docs/API-Clients.1985446128.html"
```

Comment:
Dependent on https://sagebionetworks.jira.com/browse/DOCS-101 being fixed.

Reply:
While this placeholder link is not ideal, it's better than what we have currently.